src/app/components/board.jsx

```diff
@@ -32,3 +32,46 @@ export default function Board() {
     </div>
   )
 }
+
+function checkDuplicates(cells, index){
+  indicesToCheck = Array(20).fill(null);
+  /* I need to check the row, the column and the remaining 4 cells of the 3x3 grid*/
+
+  /* first for-loop for the row*/
+  for(let r = index % 9; r < index; r++){ // Left side
+    indicesToCheck.push(r)
+  }
+  for(let r = index + 1; r < index % 9 + 8; r++){ // Right side
```

Comment:
there seems to be off by one, which leaves last cell for each row unchecked, one with index 8

Reply:
Oh I see! Thanks, fixed it now.